napari/components/viewer_model.py

```diff
@@ -658,9 +663,56 @@ def add_image(
 
             return layer_list
 
+    def open_sample(self, plugin: str, sample: str, **kwargs):
```

Comment:
Should we add `reader_plugin` as a named kwarg here? Could be useful if you know you want to open this sample with this plugin?

Reply:
added this